### PR TITLE
Reconfiguration of recording oracle to Deno based 

### PR DESCRIPTION
**Pull request by UncleTom29**

### CODE REVIEW

File: packages/examples/fortune/recording-oracle/src/contracts/EscrowAbi.json [hunk 1]
```diff
@@ -0,0 +1,453 @@
+[
```
Comment (posix4e) [inline packages/examples/fortune/recording-oracle/src/contracts/EscrowAbi.json:1]: Please do not checkin json of abi

**Review by posix4e — CHANGES_REQUESTED** [review]

**Comment by foufrix** [comment]
Hi, after trying out your repo, I can setup the escrow, but when I send the first message I get an axios error :
image

The two first with uncaught promise come from my extension, so you can neglect them

Those are the one important : 
image

**Review by posix4e — CHANGES_REQUESTED** [review]
Stop checking in Escrow.json. That is not ok

File: packages/examples/fortune/recording-oracle/hardhat.config.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-/** @type import('hardhat/config').HardhatUserConfig */
-module.exports = {
-  solidity: '0.6.2',
-};
+/** @type import('hardhat/config').HardhatUserConfig */
```
Comment (posix4e) [inline packages/examples/fortune/recording-oracle/hardhat.config.js:1]: why are you reformatting this file?

File: packages/examples/fortune/recording-oracle/LICENSE [hunk 3]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (posix4e) [inline packages/examples/fortune/recording-oracle/LICENSE:1]: Why are you checking in GPL? That's not ok

**Comment by UncleTom29** [comment]
Sorry for that, it was used to test. I’ll remove it now
Moreover during deployment, there’s no way to mirror the NPM workspace init
in Deno. So there’s no way to import the ***@***.***/core” module. It
might cause the Deno app not to run properly.

**Comment by UncleTom29** [comment]
@posix4e requested changes implemented

**Pull request closed without merging**